examples/datadog/agent_otlp.yaml

```diff
@@ -0,0 +1,24 @@
+## Datadog Agent with OTLP enabled
+
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  tags: []
+  apm:
+    # Enable APM over Socket (Unix Socket or windows named pipe)
+    socketEnabled: true
+    socketPath: /var/run/datadog/apm.socket
+    hostSocketPath: /var/run/datadog/
+  ## process-agent is not required for apm collection
+  otlp:
+    receiver:
+      protocols:
+        grpc:
+          enabled: false
+          endpoint: "0.0.0.0:4317"
+          useHostPort: true
+        http:
+          enabled: false
+          endpoint: "0.0.0.0:4318"
+          useHostPort: true
```

Comment:
For otlp these (`...grpc.enabled` and `...http.enabled`) can be set to `true`. To ensure the Agent is listening for them.